src/sentry/static/sentry/app/views/organizationIntegrations/providerRow.tsx

```diff
@@ -1,4 +1,3 @@
-import {Box, Flex} from 'reflexbox';
 import {withTheme} from 'emotion-theming';
 import PropTypes from 'prop-types';
```

Comment:
@NisanthanNanthakumar We should definitely not start modifying existing components which could introduce bugs into the existing flow. Let's just make a new version of this component that we can use for the new directory. We can consolidate the components later if it makes sense.

Reply:
@scefali ESLint doesn't let me commit without refactoring reflexbox out of the file.

Reply:
@NisanthanNanthakumar you can always use `-n`. But also, you should definitely refactor in the new component you make.